src/assets/locales/de/main.json

```diff
@@ -65,6 +65,7 @@
     "admin-credential": "Certification Mgmt",
     "companyCertificate": "Company Certificates",
     "companyWallet": "Company Wallet",
+    "company-subscriptions": "vFirmenabonnements",
```

Comment:
I assume the v should not be there, only Firmenabonnements or Unternehmensabonnements